tests/lib/prepare.sh

```diff
@@ -881,12 +881,24 @@ EOF
 
 # prepare_ubuntu_core will prepare ubuntu-core 16+
 prepare_ubuntu_core() {
+
     # we are still a "classic" image, prepare the surgery
     if [ -e /var/lib/dpkg/status ]; then
         setup_reflash_magic
         REBOOT
     fi
 
+    # XXX: we've just rebooted and cloud-init created a broken account for the
+    # "ubuntu" user.  The account has a home directory /home/ubuntu but is
+    # actually owned by root:root.
+    # Open questions:
+    # - Why does this happen, is it cloud-init?
+    # - Why is the ownership wrong?
```

Comment:
Can this be a separate PR?

Reply:
I you feel strongly about it, sure. Is there a particular worry you want to address here?

Reply:
My worry is that we don't know where does it come from and this hides the problem. So I'm not convinced we should land this change (while the rest of this PR is uncontroversial).

Reply:
We know this is coming from cloud-image. Beyond that it's something to debug

Reply:
I've dropped this now. We just ignore the /home/ubuntu ownership.